lib/index.js

```diff
@@ -5,7 +5,63 @@ const spawn = require('child_process').spawn;
 const spawnSync = require('child_process').spawnSync;
 const JSON = require('./json-buffer');
 const isElectron = require('is-electron');
+const fs = require('fs');
+const os = require('os');
+const index = [0];
+const timeStamp = new Date().toISOString();
+const net = require('net');
+let clientConnected = false;
+let client;
+const READFILETIMELIMIT = 5000;
+const debugMode = false;
```

Comment:
what is debugMode?

Reply:
When we need to debug this lib again, setting it to `true` will enable all the existing logging. Don't want to re-implement them...